vulnerabilities/tests/test_api.py

```diff
@@ -194,95 +194,73 @@ def test_package_serializer(self):
 
 
 class TestBulkAPIResponse(TestCase):
-    fixtures = ["debian.json"]
-
-    def test_bulk_vulnerabilities_api(self):
-        request_body = {"vulnerabilities": ["CVE-2009-1382", "CVE-2014-8242", "RANDOM-CVE"]}
-        expected_response = {
-            "CVE-2009-1382": {
-                "resolved_packages": [
-                    OrderedDict(
-                        [
-                            ("url", "http://testserver/api/packages/2"),
-                            ("purl", "pkg:deb/debian/mimetex@1.74-1?distro=jessie"),
-                        ]
-                    ),
-                    OrderedDict(
-                        [
-                            ("url", "http://testserver/api/packages/3"),
-                            ("purl", "pkg:deb/debian/mimetex@1.50-1.1?distro=jessie"),
-                        ]
-                    ),
-                ],
-                "unresolved_packages": [],
-                "url": "http://testserver/api/vulnerabilities/2",
-            },
-            "CVE-2014-8242": {
-                "resolved_packages": [],
-                "unresolved_packages": [
-                    OrderedDict(
-                        [
-                            ("url", "http://testserver/api/packages/1"),
-                            ("purl", "pkg:deb/debian/librsync@0.9.7-10?distro=jessie"),
-                        ]
-                    )
-                ],
-                "url": "http://testserver/api/vulnerabilities/1",
-            },
-            "RANDOM-CVE": {},
-        }
-
-        response = self.client.post(
-            "/api/vulnerabilities/bulk_search/", data=request_body, content_type="application/json"
-        ).data
-        assert response == expected_response
+    fixtures = ["github.json"]
 
     def test_bulk_packages_api(self):
         request_body = {
-            "packages": [
-                "pkg:deb/debian/librsync@0.9.7-10?distro=jessie",
-                "pkg:deb/debian/mimetex@1.50-1.1?distro=jessie",
+            "purls": [
+                "pkg:deb/debian/doesnotexist@0.9.7-10?distro=jessie",
+                "pkg:maven/com.datadoghq/datadog-api-client@1.0.0-beta.7",
             ]
         }
         response = self.client.post(
-            "/api/packages/bulk_search/", data=request_body, content_type="application/json"
-        ).data
-        expected_response = {
-            "pkg:deb/debian/librsync@0.9.7-10?distro=jessie": {
-                "resolved_vulnerabilities": [],
+            "/api/packages/bulk_search/",
+            data=request_body,
+            content_type="application/json",
+        ).json()
+
+        expected_response = [
+            {},
```

Comment:
What is this? IMHO it would be better to return an actual data structure like below, with no vulnerabilities

Reply:
Done